packages/react-components/src/components/styles/v8StyleShim.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IStyle } from '@fluentui/react';
+import { ComponentSlotStyle } from '../../types';
+
+/**
+ * Function to create a React.CSSProperties object from a v8 style object.
+ * This function is still not ideal
+ * as v8Style can use pseudo-class selectors that style objects can't process correctly.
```

Comment:
Just making a note to try remember to bug bash this and what the impact would be